README.md

```diff
@@ -161,6 +161,14 @@ The IPinfo library can be authenticated with your IPinfo API token, which is pas
 
 In-memory caching of `details` data is provided by default via the [cachetools](https://cachetools.readthedocs.io/en/latest/) library. This uses an LRU (least recently used) cache with a TTL (time to live) by default. This means that values will be cached for the specified duration; if the cache's max size is reached, cache values will be invalidated as necessary, starting with the oldest cached value.
 
+You can access the cached property to see if the response was `cached`.
```

Comment:
The wrong `cached` was wrapped with '`'; in the example at https://github.com/ipinfo/python/pull/54#discussion_r664774261 I wrapped only the first one.